### DISCUSSION TITLE
Definition of ASE stress vs deepmd virial

### DISCUSSION THREAD

**Discussion by romankempt** [deepmodeling/deepmd-kit#1991] [Q&A] [answered]
Hi, I'm not familiar with the key difference between the stress in voigt notation and the virial tensor.

I've noticed that here [in the dpdata ase plugin](https://github.com/deepmodeling/dpdata/blob/2ac2f48b6761fb9b53236edcb533b3dc95ee264e/dpdata/plugins/ase.py#L88-L94), they differ by a factor of the negative volume.

So if I wish to parse data from the ASE to a deepmd LabeledSystem, do I have to define the virial as:
```
virials = atoms.get_stress(voigt=False)
```

or 
```
stress = atoms.get_stress(voigt=False)
virials = np.array([-atoms.get_volume() * stress]).astype('float32')
```

Thanks in advance!

**Comment by njzjz** [comment, marked as answer]
Yes, you can see discussion in #338.

**Reply by romankempt** [reply]
I see, thanks! :)
